.github/workflows/build-and-test.yml

```diff
@@ -27,11 +27,3 @@ jobs:
     with:
       xcodeprojname: Tests/UITests/UITests.xcodeproj
       scheme: TestApp
-  create-and-upload-coverage-report:
```

Comment:
Temporarily disabling this step as it is throwing an unclear error. cc: @PSchmiedmayer

Reply:
The problem you are facing is related to the artifact and input names:
The `.xcresult` bundle that the SPM task generates and uploads is called `HealthKitOnOMH-Package.xcresult`: https://github.com/StanfordBDHG/HealthKitOnOMH/actions/runs/3666766668/jobs/6198746770#step:6:23

The input that you describe below is named `HealthKitOnOMH.xcresult`. This is an unfortunate problem as the scheme for xcodebuild changes from `HealthKitOnOMH` to `HealthKitOnOMH-Package` depending on how many targets you have in your Swift package.

Therefore the proposed change should fix your problem:
```
  create-and-upload-coverage-report:
    name: Create and Upload Coverage Report
    needs: [build_and_test-spm, build_and_test-uitests]
    uses: StanfordBDHG/.github/.github/workflows/create-and-upload-coverage-report.yml@v1
    with:
      coveragereports: HealthKitOnOMH-Package.xcresult HealthKitOnOMH.xcresult
    secrets:
      token: ${{ secrets.CODECOV_TOKEN }}
```

Reply:
This makes sense, thank you @PSchmiedmayer!

Reply:
You're welcome 🚀👍